lib/cli/blueprints/app/files/__name__/.travis.yml

```diff
@@ -0,0 +1,19 @@
+---
+language: node_js
+node_js:
+  - "6"
+
+sudo: false
+
+cache:
+  directories:
+    - node_modules
+
+before_install:
+  - npm config set spin false
+
+install:
+  - npm install
+
+script:
```

Comment:
install and script are redundent I think since travis does that by default with node, also the cache most likely too.